misc/python/materialize/buildkite_insights/step_durations/analysis.py

```diff
@@ -175,7 +175,7 @@ def print_stats(
 
 def main(
     pipeline_slug: str,
-    build_step_keys: list[str],
+    build_step_keys: list[tuple[str, int | None]],
```

Comment:
Can you add a comment here? It is unclear to me what this parameter now holds.

Reply:
Sure, let me change the type instead to make it clearer.